api/functions/updateDbUsers.js

```diff
@@ -0,0 +1,41 @@
+export const updateDbUsers = async (usersFile, db) => {
+	try {
+		await db.query("BEGIN");
+
+		await db.query("DELETE FROM all_users");
```

Comment:
Rather than deleting all users and then reinserting them again, you can use the concept of an 'upsert'.

I'm not an expert on postgresql but [this article](https://www.geeksforgeeks.org/postgresql-upsert/) explains the concepts